templates/acs.yaml

```diff
@@ -242,6 +248,7 @@ Resources:
                     - !Sub "--s3bucket-kms-alias ${S3BucketKMSAlias}"
                     - !Sub "--s3bucket-location ${AWS::Region}"
                     - !If [PrivateRegistry, !Join ["", [--registry-secret, " ", !Ref RegistryCredentials]], !Ref "AWS::NoValue"]
+                    - !Sub "--yourkit-opts ${YourKitAgentOps}"
```

Comment:
This makes the YourKit parameters mandatory, it would be good to follow the same pattern as the previous line and check for empty string before adding the option, this would allow users to omit the YourKit options if they wanted.